### ISSUE TITLE
Keep the stream/video going even after you close the CMD window.

### ISSUE THREAD

**Issue by dragons4life** [chrippa/livestreamer#1198]

**Comment by ghost** [comment]
[`--player-passthrough`](http://docs.livestreamer.io/cli.html#cmdoption--player-passthrough)

Note: This might not work with all streams. It depends which type of stream you are trying to access and which video player you are using.

**Comment by dragons4life** [comment]
What's the default livestreamer uses?

hls
http
rtmp

also thanks.

**Comment by ghost** [comment]
You have to specify the stream type by yourself, or you simply use all possible stream types at once.

```
--player-passthrough http,hls,rtmp
```

You can also edit the [configuration file](http://docs.livestreamer.io/cli.html#configuration-file).

```
--player-passthrough=http,hls,rtmp
```

**Comment by dragons4life** [comment]
Yeah I know, so far http closes the player after you close the CMD, unlike HLS.

but I wanted to know which one livestreamer uses it as default (fresh install), guessing is http

**Comment by ghost** [comment]
[`--player-passthrough`](http://docs.livestreamer.io/cli.html#cmdoption--player-passthrough) option is disabled by default. As I said before, look at your configuration file ... - or submit a proper command line switch (e.g. --player-passthrough http).
